### DISCUSSION TITLE
Any other solutions similar to privateGPT?

### DISCUSSION THREAD

**Discussion by alejandropascual12** [imartinez/privateGPT#856] [General]
Conducting some research, wanted to reach out to the community first and gather some insights.

Are there other similar solutions to privateGPT that provide a private LLM and prevent data leakage? Although privateGPT doesn't require an internet connection, are there any open-source solutions that are scalable and may allow for internal corporate communications that would _not_ be local? Essentially, I'm trying to see if there are any private LLMs that can be implemented within an enterprise environment, - or if privateGPT is capable of doing so already.

Thanks for reading.

**Comment by Alshamaa13** [comment]
Try GPT4All
